.github/workflows/release.yaml

```diff
@@ -69,14 +71,18 @@ jobs:
           mkdir -p .tmp/bin
           cp ./protobuf-${{ needs.get-protobuf.outputs.version }}/bazel-bin/conformance/conformance_test_runner .tmp/bin/conformance_test_runner
 
-      - name: Run Conformance Tests
+      - name: Verify Binary
         run: |
-          make testci
+          .tmp/bin/conformance_test_runner --help 2>&1 | grep 'Usage: conformance-test-runner \[options\] <test-program>' || { echo "Build looks broken"; exit 1; }
+
+      - name: Copy Protos
+        run: |
+          cp -r proto ./tmp/include
```

Comment:
If proto files and conformance runner get out of sync, users will get _very_ difficult to debug test failures.

You _have_ to use the proto files from release you just downloaded, not from this repo.

Reply:
Got it. Updated.